docs/preview/03-Features/01-Azure/01-service-bus.mdx

```diff
@@ -6,15 +6,15 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Azure Service Bus messaging
-The `Arcus.Messaging.Pumps.ServiceBus` library provides a way to process Azure Service Bus messages on queues/topic subscriptions via custom routed *message handlers*, instead of interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver) yourself. 
+The `Arcus.Messaging.ServiceBus` library provides a way to process Azure Service Bus messages on queues/topic subscriptions via custom routed *message handlers*, instead of interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver) yourself. 
```

Comment:
I'd change this sentence a bit, as there are multiple ways to retrieve messages from ServiceBus.

What about:

```suggestion
The `Arcus.Messaging.ServiceBus` library provides a way to process Azure Service Bus messages on queues/topic subscriptions using *message handlers*.  The library takes care of retrieving messages from Azure Service Bus and route them to the correct *message handler* which will process the message.  This means you don't have to worry about retrieving the messages from Service Bus yourself. 
```